src/vs/platform/windows/electron-main/windowImpl.ts

```diff
@@ -966,7 +966,8 @@ export class CodeWindow extends BaseWindow implements ICodeWindow {
 				newHttpProxy = newHttpProxy.substr(0, newHttpProxy.length - 1);
 			}
 
-			const newNoProxy = (process.env['no_proxy'] || process.env['NO_PROXY'] || '').trim() || undefined; // Not standardized.
+			const newNoProxy = (this.configurationService.getValue<string[]>('http.noProxy') || ['']).map((item) => item.trim()).join(",")
```

Comment:
Use single quotes.

Reply:
Thanks for the info, I have changed double quotes to single quotes

https://github.com/microsoft/vscode/blob/d5b121787ac56b507385fdb81e002fc938d67f5d/src/vs/platform/windows/electron-main/windowImpl.ts#L969